yearn/apy/curve/simple.py

```diff
@@ -87,6 +87,23 @@ def simple(vault, samples: ApySamples) -> Apy:
         reward_address = gauge.reward_contract()
         if reward_address != ZERO_ADDRESS:
             reward_apr = rewards(reward_address, pool_price, base_asset_price, block=block)
+    elif hasattr(gauge, "reward_data"): # this is how new gauges, starting with MIM, show rewards
+        # get our token
+        gauge_reward_token = gauge.reward_tokens(0)
+        
+        # get our period end
+        period_finish = gauge.reward_data(gauge_reward_token)[2]
+        
+        # get our total supply
+        _distributor = gauge.reward_data(gauge_reward_token)[1]
+        token_contract = Contract(gauge_reward_token)
```

Comment:
getting an error here - maybe wrap this line and the ones below in a try/except?